app/bundles/LeadBundle/Views/Lead/lead.html.php

```diff
@@ -205,7 +205,11 @@
                                                         <?php if (is_array($field['value']) && 'multiselect' === $field['type']): ?>
                                                             <?php echo implode(', ', $field['value']); ?>
                                                         <?php else: ?>
-                                                            <?php echo $field['value']; ?>
+                                                            <?php if ('select' === $field['type']): ?>
```

Comment:
I could do it monday, yes, but I tried keeping it consistent already by doing it like the third line above: ```<?php if (is_array($field['value']) && 'multiselect' === $field['type']): ?>```
But I completely agree with you!